pkgs/development/ocaml-modules/tcslib/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ocamlPackages, buildOasisPackage }:
```

Comment:
Each OCaml library should be a different argument: `ounit`, `ocaml_extlib` and `num`; rather than the whole set `ocamlPackages`. Otherwise it will not be possible to build the package for other versions of OCaml.

Reply:
Oh OK I thought that would be handled further up in the path e.g. `ocamlPackages` already containing the package set for the "correct" version